notebooks/0370_infill-scenarios-leftovers.py

```diff
@@ -162,8 +162,8 @@ def transform_rcmip_to_iamc_variable(v):
     "Emissions|HFC|HFC365mfc": "Emissions|HFC|HFC134a",
     "Emissions|Montreal Gases|CH2Cl2": "Emissions|HFC|HFC134a",
     "Emissions|Montreal Gases|CHCl3": "Emissions|C2F6",
-    # "Emissions|Montreal Gases|CH3Br": "Emissions|C2F6",
-    # "Emissions|Montreal Gases|CH3Cl": "Emissions|CF4",
+    "Emissions|Montreal Gases|CH3Br": "Emissions|C2F6",
+    "Emissions|Montreal Gases|CH3Cl": "Emissions|CF4",
     "Emissions|NF3": "Emissions|SF6",
     "Emissions|Montreal Gases|Halon1202": "Emissions|Montreal Gases|Halon1211",
```

Comment:
For these in general, do we want to document the reasons?

Reply:
Yep if I had them I would. At the moment this is legacy from CMIP6. At some point I'll get hold of Luke Western and some other experts to make a better decision (I'm not that worried right now, these gases have small influence).